### DISCUSSION TITLE
Config Help Needed

### DISCUSSION THREAD

**Discussion by xyeoda** [Clooos/Bubble-Card#309] [Feature requests]
Been setting up my new dashboard …what an amazing piece of work!

i have two issues because i had multiple rooms defined in my horizontal stack i widened it to 800px.

1) the pop up is not centred any more:
<img width="936" alt="Screenshot 2023-11-13 at 8 46 56 PM" src="https://github.com/Clooos/Bubble-Card/assets/31374940/417c8231-4829-43a6-a973-5ae8e22bb86a">

2) one of the pop-up seems to block my horizontal stack - while the rest are just fine.
<img width="977" alt="Screenshot 2023-11-13 at 8 47 03 PM" src="https://github.com/Clooos/Bubble-Card/assets/31374940/7abb450a-b3ed-4716-89ca-f31ae586c734">

**Comment by Clooos** [comment]
Hi and thanks! 

For your first issue, this is only because you don't have "enough" buttons in your horizontal buttons stack but this is indeed something that I need to fix.

And for the second you just need to put your horizontal buttons stack as the last card on your view 🙂
